checkov/terraform/checks/resource/aws/EMRClusterIsEncryptedKMS.py

```diff
@@ -16,7 +16,7 @@ def scan_resource_conf(self, conf):
         if 'configuration' not in conf:
             return CheckResult.SKIPPED
         configuration = conf['configuration'][0]
-        if "SSE-KMS" in configuration:
+        if "SSE-KMS" in str(configuration):
```

Comment:
ok, this is needed, because it is now a `dict`? could change it to a `BaseValueResourceCheck`? then the inspected key should be something like `configuration. EncryptionConfiguration.AtRestEncryptionConfiguration. S3EncryptionConfiguration. EncryptionMode` and expected value `SSE-KMS`

Reply:
Yeah it's beacuse it'ss a dict :)
I actually checked it out but I thought there might be more places to place `SSE-KMS` in (maybe not this field alone I mean). If there are no more places I will change it